client/src/components/MainPage/TranslationsPage/DonateModal/AudioTranslation/index.jsx

```diff
@@ -36,6 +72,19 @@ class AudioTranslation extends Component {
         <h3 className="file__name" ref="fileName">
           No file Choosen
         </h3>
+        <hr />
+        <h4>Or record audio</h4>
+        <FontAwesomeIcon
+          icon={faMicrophone}
+          className={`record-button ${recording ? 'recording' : null}`}
+          onClick={this.startRecord}
+        />
+        <FontAwesomeIcon
+          icon={faStop}
+          className={`stop__button ${!recording ? 'stop__button--disabled' : null}`}
```

Comment:
@HemaSAli 
the [classnames](https://www.npmjs.com/package/classnames) npm package helps with writing these conditional class names, if you want to give it a try.
Then this line would become:
`className={classnames('stop__button', {'stop__button--disabled': recording})}`
and line 79 would become:
`className={classnames('record-button', {recording})}`

Reply:
@tomduggan85, at first thank you Tom for your participation with us here
I really don't know about this npm package, I will edit my code and use it.

Reply:
Cool - it makes my life a lot easier!  I think you'll like it.

Reply:
yeah yeah, In fact it took from me a time to write this line :laughing: 
```
{`stop__button ${!recording ? 'stop__button--disabled' : null}`}
```